documentation-site/components/walkthrough.js

```diff
@@ -0,0 +1,210 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+import * as React from 'react';
+import {
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  ModalButton,
+  SIZE,
+  ROLE,
+} from 'baseui/modal';
+import {StatefulTabs, Tab} from 'baseui/tabs';
+import {withStyle, useStyletron} from 'baseui';
+import {Button} from 'baseui/button';
+
+import {Label3, Label4, Paragraph3} from 'baseui/typography';
+import {StatefulTooltip} from 'baseui/tooltip';
+
+import CookiesConstructor from 'universal-cookie';
+const Cookies = new CookiesConstructor();
```

Comment:
This should probably be inside of the component. Otherwise the `cookie` value never changes unless you reload.

Reply:
this should solve it https://github.com/uber/baseweb/pull/2199/files#diff-12eed97910803029f83c0e98ee55f26aR35